tty-progressbar.gemspec

```
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "strings-ansi", "~> 0.2"
   spec.add_dependency "tty-cursor", "~> 0.7"
   spec.add_dependency "tty-screen", "~> 0.8"
-  spec.add_dependency "unicode-display_width", ">= 1.6", "< 3.0"
+  spec.add_dependency "unicode-display_width", ">= 2.4.0", "< 4.0"
```

This changes both the lower and upper constraints. However, for backwards compatibility with older versions of Ruby, the lower constraints should be left intact.

```suggestion
  spec.add_dependency "unicode-display_width", ">= 1.6", "< 4.0"
```